README.md

```diff
@@ -1,38 +1,6 @@
-![cf](https://i.imgur.com/7v5ASc8.png) Lab 20 - Intro to Angular and Webpack
-======
-
-## To Submit this Assignment
-  * create a fork of this repository
-  * write all of your code in a directory named `lab-` + `<your name>` **e.g.** `lab-brian`
-  * push to your repository
-  * submit a pull request to this repository
-  * submit a link to your PR in canvas
-  * write a question and observation on canvas
-
-## Include
-  * `.eslintrc`
-  * `.gitignore`
-  * `package.json`
-    * create an npm `watch` script for running `webpack-dev-server --inline --hot`
-    * create an npm `lint` script for linting your JS with `eslint`
-  * **ignore the build directory**
-  * `webpack.config.js`
+# Lab-26 Cowsay-app *meowsay*
 
 ## Description
```

Comment:
**TODO FOR LOST POINTS**

Good README, but remember the instructions said ``write a description of what angular modules, components and controllers are for``.